07LessonSeven/in-class-project/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+
+		<title>NASA</title>
+		<link rel="stylesheet" href="css/my-style.css">
+
+	</head>
+
+	<body>
+		<header>
+
+			<!--White font color in middle of image  -->
+
```

Comment:
because this is the main header of our page, try making this an h1. also, in your stylings, make the margin=0. the extra space at the top of your page was coming from default margin.